### DISCUSSION TITLE
what would relevant Hunger Games notifications look like ?

### DISCUSSION THREAD

**Discussion by teolemon** [openfoodfacts/hunger-games#1058] [Q&A]
### Problem
- People forget about Hunger Games
- As we embark on relevant notifications across Open Food Facts, what would relevant Hunger Games notifications look like ? Assume that we can leverage Robotoff, and various Events, possibly data about contributor edits (like favorite categories where the user validated insights) but no personal data

**Comment by teolemon** [comment]
@alexfauquette @raphael0202 @VaiTon

**Comment by alexfauquette** [comment]
In the question game, we have the star to save questions.

We could send a notification with the evolution of the number of questions for the filter they saved

![image](https://github.com/user-attachments/assets/b7779263-eba4-4207-b4ec-cf0f52745094)
